hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SecurityUtil.java

```diff
@@ -786,4 +789,96 @@ public static List<ZKUtil.ZKAuthInfo> getZKAuthInfos(Configuration conf,
       throw e;
     }
   }
+
+  public static void validateSslConfiguration(TruststoreKeystore truststoreKeystore) throws ConfigurationException {
+    if (org.apache.commons.lang3.StringUtils.isEmpty(truststoreKeystore.keystoreLocation)) {
```

Comment:
would be nice to import org.apache.commons.lang3.StringUtils in the beginning and get rid of the prefix in the code. I believe all the old commons-lang library code is removed from Hadoop now we we only use commons-lang3.

Reply:
I thought of this too, but in this class we already have `org.apache.hadoop.util.StringUtils` imported and it is using `StringUtils.toLowerCase()` and `StringUtils.toUpperCase` from there. This library doesn't have the `isEmpty()` method and the `org.apache.commons.lang3.StringUtils` doesn't have the other two (at least they have a different name). 

What I could do is to use the `commons-lang3` and use the `upperCase()` and `lowerCase()` methods from there, but I didn't want to do unnecessary refactoring. 

Or I can check this here and not use `commons-lang3`:
```
truststoreKeystore.keystoreLocation == null || truststoreKeystore.keystoreLocation.isEmpty()
```

These were my first thoughts, let me know if you think something else could be done.

Reply:
ok got it. i have no problem with that.